core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -172,7 +172,7 @@
             </li>
             <li class="nav-item">
               <a class="dropdown-item nav-link oppia-navbar-tab-content"
-                 href="https://www.oppiafoundation.org/partnerships"
+                 href="/partnerships"
```

Comment:
@iamprayush I guess we also need to change the href in the side navigation bar. PTAL

Reply:
Thanks for catching that @ashutoshc8101! I'll fix it in my next PR.